proto/proto/devices.pb.go

```diff
@@ -1,19 +1,20 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.0
-// 	protoc        v3.21.9
+// 	protoc        v3.12.4
```

Comment:
Please use the Docker images as mentioned in the `README.md` file to regenerate these proto files so that we can keep the same versions as before:
```
docker build -f proto/Dockerfile --target proto-js -t wg-access-server-proto:js .
docker build -f proto/Dockerfile --target proto-go -t wg-access-server-proto:go .
docker run --rm -v `pwd`/proto:/proto -v `pwd`/website/src/sdk:/code/src/sdk wg-access-server-proto:js
docker run --rm -v `pwd`/proto:/code/proto wg-access-server-proto:go
```